introduction.md

```diff
@@ -0,0 +1 @@
+This is an introduction about me. I like to retain an air of mystery so I won't say any more
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.